### PR TITLE
JZ -  added Leaderboard Feature for totalWealth to be displayed in the desired format

### PR DESCRIPTION
**Pull request by jczhou02**
Additionally, made changes to   [frontend/src/tests/components/Leaderboard/LeaderboardTable.test.js]
closes issue #11   

[Storybook](https://ucsb-cs156-m23.github.io/proj-happycows-m23-10am-2/prs/16/storybook/?path=/story/components-leaderboard-leaderboardtable--five-user-commons-admin)

Screenshot (264)

### CODE REVIEW

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js
Comment (rockygao2020) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:1]: Code checks out with the required changes, and the tests make perfect sense.

**Review by rockygao2020 — DISMISSED** [review]
Code checks out with the required changes, and the tests make perfect sense.

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
         {
             Header: 'Total Wealth',
             accessor: 'totalWealth',
+            //this is the func to  can return custom tableCell
+            Cell : (props)=>{
+            //props.value will contain your date
+            //you can convert total wealth data here
+                const custom_wealth = '$'+props.value.toFixed(2)
+                return <span>{custom_wealth}</span>
```
Comment (pconrad) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:24]: If you do this, it should display right aligned, which makes more sense for currency.  

```suggestion
                return div style={{textAlign: "right"}}>{custom_wealth}</div>
```

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js [hunk 2]
```diff
@@ -16,6 +16,13 @@ export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
         {
             Header: 'Total Wealth',
             accessor: 'totalWealth',
+            //this is the func to  can return custom tableCell
```
Comment (pconrad) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:19]: I suggest removing this comment.

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js [hunk 3]
```diff
@@ -16,6 +16,13 @@ export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
...
+            Cell : (props)=>{
+            //props.value will contain your date
+            //you can convert total wealth data here
```
Comment (pconrad) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:22]: I suggest removing this comment
```suggestion
```

File: frontend/src/main/components/Leaderboard/LeaderboardTable.js (hunk 2 above)
Comment (pconrad) [inline frontend/src/main/components/Leaderboard/LeaderboardTable.js:19]:
```suggestion
```

**Review by pconrad — CHANGES_REQUESTED** [review]
LGTM... but I suggest removing a few unnecessary comments, as well as right aligning the dollar amounts so that they all line up in the table column.

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**